### PR TITLE
Viewability search form

### PR DESCRIPTION
**Pull request by uplne**
1. Removing close on scroll
2. Detecting mobile
3. Disabling autoComplete
4. Updated behaviour

Dekstop in normal mode:
searchdesktop1

Desktop in slim mode:
searchdesktop2

Mobile:
searchmobile

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/navigation/search.js [hunk 1]
```diff
@@ -48,9 +56,10 @@ define([
         }
 
         this.focusSearchField = function () {
-            var $input = $('input.gsc-input');
+            var $input = $('input.gsc-input'),
+                inputLenght = $input.length;
```
Comment (robertberry) [inline static/src/javascripts/projects/common/modules/navigation/search.js]: `inputLength`

Comment (robertberry) [inline static/src/javascripts/projects/common/modules/navigation/search.js]: Also why are you putting this into a `var`? You only seem to use it in one place.

**Comment by robertberry** [comment]
:+1: Make sure you link it to another pull request removing an equivalent number of lines of JS

**Comment by uplne** [comment]
Challenge accepted. -29 lines in next PR it is.

**Pull request merged**